convert2rhel/unit_tests/pkghandler_test.py

```diff
@@ -1921,6 +1899,49 @@ def test_find_pkg_names_no_names(output, message):
     assert pkghandler.find_pkg_names(output, message) == set()
 
 
+class TestInstallGpgKeys(object):
+    data_dir = os.path.realpath(os.path.join(os.path.dirname(__file__), "..", "data", "version-independent"))
```

Comment:
Just a minor suggestion here: 

We could change it to be like the other examples in this PR: 

```suggestion
    data_dir = os.path.realpath(os.path.join(os.path.dirname(__file__), "../data/version-independent"))
```

No need to change this piece of code, just pointing out that we could make it more identical to the other ones.

Reply:
I'm conflicted on this one.  It's better practice to use `os.path.join()` to join paths since the path separator is different on different platforms.  However, convert2rhel is only going to be run on linux systems so we don't strictly need to do this.  And you're right that many of our current uses of `os.path.join()` have `/` hardcoded into the paths that are passed in.

So maybe we should use hardcoded `/` for now and cleanup all of of the places where `/` occurs in `os.path.joins()` as a later PR.